addon/src/lib/reducers/sideEffects.js

```diff
@@ -78,11 +76,14 @@ export function reducer(
     case actions.EXPERIMENTS_LOADED.type:
       return ({ dispatch, loader }) => {
         loader.schedule();
-        Object.keys(payload.experiments)
+        Object
+          .keys(payload.experiments)
           .map(id => payload.experiments[id])
           .filter(x => x.uninstalled && new Date(x.uninstalled) < new Date())
-          .forEach(experiment => dispatch(actions.UNINSTALL_EXPERIMENT({ experiment })))
-      }
+          .forEach(
+            experiment => dispatch(actions.UNINSTALL_EXPERIMENT({ experiment }))
```

Comment:
I'm using [prettier](https://www.npmjs.com/package/prettier) so I don't have to think about it. Use `npm run format`. I haven't automated it beyond that. Maybe its should run on `npm start`? or test?